roles/common/templates/rules.v6.j2

```diff
@@ -83,7 +83,7 @@ COMMIT
 # particular virtual (tun,tap,...) or physical (ethernet) interface.
 
 # Accept DNS traffic to the local DNS resolver
--A INPUT -d fcaa::1 -p udp --dport 53 -j ACCEPT
+-A INPUT -d {{ local_service_ipv6 }}/128 -p udp --dport 53 -j ACCEPT
```

Comment:
```suggestion
-A INPUT -d {{ local_service_ipv6 }} -p udp --dport 53 -j ACCEPT
```

Reply:
No, because `local_service_ipv6` is the destination address that we should allow to talk to, not the source

Reply:
interesting! cool, thank you.

Reply:
interesting! cool thank you